### PR TITLE
Document todos for new maintainers

### LINKED ISSUE #275: Document maintenance change from the perspective of the new maintainer
Scenario: old maintainer is gone

- [x] Get repo rights or fork
- [x] Update documentation files where maintainer is noted
    - [x] README, CITATION.cff, contributors list (all-contributors bot), ...
- [x] Check and update https://github.com/hexatomic/hexatomic/blob/develop/CONTRIBUTING.md#code-review-schedules
- [x] Prepare for development (cf. dev docs)
- [x] Check if build runs & fix if necessary
- [x] Check for open PRs
     - [x] Make release when PRs are merged
- [x] Check for open issues

### PR DESCRIPTION
**Pull request by sdruskat**
## Pull request type

 

- Documentation content changes

## What is the current behavior?

Issue Number: #275 

## What is the new behavior?

- Remove code review schedule for reasons of maintainability
- Document tasks for new maintainers to get maintenance ready
- Fix #275

## Does this introduce a breaking change?

No

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added **or** the PR is neither a bug fix nor a feature
- [x] Docs have been reviewed and added / updated if needed **or** the PR is neither a bug fix nor a feature
- [x] The *Unreleased* section in CHANGELOG.md has been amended to reflect the changes in this PR if needed
- [x] Build (`mvn verify`) was run locally and any changes were pushed
- [x] The pull request is against the correct branch (`master` for bug fixes, `develop` for new functionality)
- [x] [Dependencies and citation templates](https://github.com/hexatomic/hexatomic/tree/develop/releng/templates) have been updated where necessary **or** there are no new dependencies

---

# Checklist for maintainers

## Releases

- Do **NOT** push the green merge button on GitHub.  
Follow the latest version of the [developer/maintainer docs](https://hexatomic.github.io/hexatomic/dev/) for making releases.
- Check that license and citation information for dependencies are complete (each dependency has a folder in `/THIRD-PARTY/`).

### CODE REVIEW

File: docs/dev/src/maintenance/taking-over/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+Update the maintainer information in the relevant places:
+
+- The *Maintainer* section in `README.md` (part of the *Team* section). You may also choose to update the *Core contributors* section.
+- The list of contributors in `README.md`. List yourself for maintenance.
+- When you have fulfilled the authorship requirements for Hexatomic, also add yourself as an author to the citation metadata in `CITATION.cff`. This is a software citation metadata file. You can read about its format in the [Citation File Format documentation](https://citation-file-format.github.io/).
+
+## Update public information about code reviews
+
+The contribution guidelines in `CONTRIBUTING.md` contain information on when code reviews are likely to be performed. Change the information there as appropriate, e.g., update the timezone.
```
Comment (thomaskrause) [inline docs/dev/src/maintenance/taking-over/README.md:53]: Spelling
```suggestion
The contribution guidelines in `CONTRIBUTING.md` contain information on when code reviews are likely to be performed. Change the information there as appropriate, e.g., update the time zone.
```

File: docs/dev/src/maintenance/taking-over/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Follow the [development setup documentation](../../development/setup.md) to set up your local system and get ready for maintenance work on the codebase.
+
+## Test the state of Hexatomic
+
+Check if the [build](../../development/getting-the-source-code.md#build-hexatomic-locally) runs, and fix any issues that you encounter.
+
+## Maintain open contributions
+
+Check if there are any open pull requests for Hexatomic in the original repository.
+If you are working in a fork, consider gettin in touch with the authors of the open pull requests and ask them to put them up again against your fork.
```
Comment (thomaskrause) [inline docs/dev/src/maintenance/taking-over/README.md:71]: Typo
```suggestion
If you are working in a fork, consider getting in touch with the authors of the open pull requests and ask them to put them up again against your fork.
```

**Review by thomaskrause — APPROVED** [review]
Thanks for the write-up. I think this covers almost all points, but we might need to discuss whether we encourage maintainers of the fork to choose a different name or how they can otherwise make clear this is a fork. Having a Citations.cff file helps because original authorship and credit is not lost, but I would expect someone to at least mention the maintainer change when publishing and citing the software.

**Comment by sdruskat** [comment]
> Thanks for the write-up. I think this covers almost all points, but we might need to discuss whether we encourage maintainers of the fork to choose a different name or how they can otherwise make clear this is a fork. 

I agree, will add something to that effect.

> Having a Citations.cff file helps because original authorship and credit is not lost, but I would expect someone to at least mention the maintainer change when publishing and citing the software.

Well, if they abide by the tba authorship requirements (#316), and there has been enough change for them to warrant a new release, I'd assume that enough work has gone into the project for them to claim authorship. So adding their name to the list of names isn't such an issue (although I guess something should be said about order of authors in the list). A change in name in a fork may be additionally helpful to identify the new version.

**Comment by sdruskat** [comment]
@thomaskrause How does https://github.com/hexatomic/hexatomic/pull/315/commits/b62017a0d93e7fd3f2d3965d575ddc663997b4c2 work for you?

**Comment by thomaskrause** [comment]
> @thomaskrause How does [b62017a](https://github.com/hexatomic/hexatomic/commit/b62017a0d93e7fd3f2d3965d575ddc663997b4c2) work for you?

Looks good to me, will merge

**Pull request merged**